### PR TITLE
Add solution andFunction and unit test for it.

### PR DESCRIPTION
**Pull request by BrianNguyen420**

### CODE REVIEW

File: simpleSequential/andFunction.py [hunk 1]
```diff
@@ -3,3 +3,5 @@
 # Ex: 
 #   and(True, False) return False
 #   and(True, True)  return True
+def andFunction(boo1: bool, boo2: bool) -> bool:
```
Comment (rubikvn2100) [inline simpleSequential/andFunction.py]: Can you use `bool1` and `bool2` instead? I never see anyone in the industry use `boo`. In fact, people don't use `bool` either. Let use `var1` and `var2`,

**Comment by rubikvn2100** [comment]
Where is the unit test that you are mention?

File: simpleSequential/andFunction.py (hunk 1 above)
Comment (BrianNguyen420) [reply simpleSequential/andFunction.py]: acknowledge.

File: simpleSequential/test_andFunction [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import unittest
+from andFunction import andFunction
+
+class TestAndFunction(unittest.TestCase):
```
Comment (rubikvn2100) [inline simpleSequential/test_andFunction]: Do you think 2 tests are enough? How about 4 to cover all cases?

Comment (BrianNguyen420) [reply simpleSequential/test_andFunction]: I think 2 is good in this problem.

File: simpleSequential/test_andFunction.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import unittest
+from andFunction import andFunction
+
+class TestAndFunction(unittest.TestCase):
+    def test_sameBoolInput(self):
```
Comment (rubikvn2100) [inline simpleSequential/test_andFunction.py]: The test name "sameBoolInput" did not reflect the test that you are using. You are passing "True, False" into the functions.

File: simpleSequential/test_andFunction.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+        self.assertEqual(andFunction(True, False), False)
+    
+    def test_diffBooInput(self):
```
Comment (rubikvn2100) [inline simpleSequential/test_andFunction.py]: The test name "diffBoolInput" did not reflect the test that you are using. You are passing "True, True" into the functions.

**Comment by rubikvn2100** [comment]
Please fix this. Make the commit name something like "Fix the test descriptions for <fileName>.<extension>"

**Pull request merged**